OrbitClientProtos/capture_data.proto

```diff
@@ -59,6 +59,7 @@ message TracepointEventInfo {
   int64 time = 3;
   int32 cpu = 4;
   uint64 tracepoint_info_key = 5;
+  bool is_in_target_process = 6;
```

Comment:
I don't think you need this. You should be able to do
`if (tracepoint.pid() == GOrbitApp->GetCaptureData().process_id())`.